### ISSUE TITLE
[Question] Is tree-shaking working?

### ISSUE THREAD

**Issue by PiotrWpl** [dint-dev/universal_io#46]
Hi,

A month ago, there was a discussion between Andrea and the Flutter team on X.com about not using the UniversalPlatform package and instead choosing kIsWeb and TargetPlatform. The concern was that using UniversalPlatform could prevent tree-shaking. So, I have a question: Is tree-shaking working fine with Universal.io, or is it the same situation where we are losing tree-shaking because of it?

Thank you for all your contributions to the Flutter Community!

Source:
- original post -> https://x.com/biz84/status/1800520573346566582
- correction -> https://x.com/biz84/status/1801161442118242783

**Comment by terrier989** [comment]
Yes. Using `kIsWeb` and `TargetPlatform` is much better in Flutter applications than using this package.
